include/exadg/structure/user_interface/parameters.cpp

```diff
@@ -159,6 +160,7 @@ Parameters::print_parameters_physical_quantities(dealii::ConditionalOStream cons
   if(problem_type == ProblemType::Unsteady)
   {
     print_parameter(pcout, "Density", density);
+    print_parameter(pcout, "Weak damping coefficient", weak_damping_coefficient);
```

Comment:
In case nonlinear problems should not be supported right now, this parameter should only be printed in cases where it is relevant.